src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1086,9 +1086,26 @@ class misc_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleSaveCommand(ChatHandler*  /*handler*/, char const* /*args*/)
+    static bool HandleSaveCommand(ChatHandler* handler, char const* /*args*/)
     {
-        // pussywizard: fully disabled on 28.12.2011, but disabled it "silently"
+        Player* player = handler->GetSession()->GetPlayer();
+
+        // save GM account without delay and output message
+        if (handler->GetSession()->GetSecurity() >= SEC_GAMEMASTER)
+        {
+            if (Player* target = handler->getSelectedPlayer())
+                target->SaveToDB(true, false);
+            else
+                player->SaveToDB(true, false);
+            handler->SendSysMessage(LANG_PLAYER_SAVED);
+            return true;
+        }
+
+        // save if the player has last been saved over 20 seconds ago
+        uint32 saveInterval = sWorld->getIntConfig(CONFIG_INTERVAL_SAVE);
+        if (saveInterval == 0 || (saveInterval > 20 * IN_MILLISECONDS && player->GetSaveTimer() <= saveInterval - 20 * IN_MILLISECONDS))
+            player->SaveToDB(true, false);
```

Comment:
can you please add brackets to all these IFs ?

Reply:
Ye when I got time